Lib9c/Delegation/RewardBase.cs

```diff
@@ -25,28 +25,47 @@ public class RewardBase : IBencodable, IEquatable<RewardBase>
 
         public RewardBase(
             Address address,
+            long startHeight,
             BigInteger totalShares,
             IEnumerable<Currency> currencies)
             : this(
                   address,
+                  startHeight,
                   totalShares,
                   currencies.Select(c => c * 0),
                   RecommendedSigFig(totalShares))
         {
         }
 
+        public RewardBase(
+            Address address,
+            long startHeight,
+            BigInteger totalShares,
+            IEnumerable<Currency> currencies,
+            int sigFig)
+            : this(
+                  address,
+                  startHeight,
+                  totalShares,
+                  currencies.Select(c => c * 0),
```

Comment:
What is intended for `c * 0`? I guess manipulating data without any explanation makes it hard to fix or refactor later.

Reply:
I've added some explanations as a comment.
Also, I've filled docstrings for `RewardBase`.